content/services/198-cal-photofusions-galleries-11456.md

```diff
@@ -3,6 +3,11 @@ organisation: 198 Cal/& Photofusion’s Galleries
 title: Artist Minds With 198 Contemporary Arts And Learning And Photofusion
 shortDescription: Photography and crafts sessions
 image:
+<<<<<<< HEAD
```

Comment:
This is a merge conflict marker, this will need to be removed from all of the files. The `=======` and `>>>>>>> development` are also markers that need to be taken out.